dronuts-frontend/src/pages/Order.tsx

```diff
@@ -1,18 +1,146 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
 // @ts-nocheck
 
-import React from "react";
-import Checkout from "../components/Checkout";
+import * as React from 'react';
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import Button from '@mui/material/Button';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import AddressForm from '../components/AddressForm';
+import TimeEstimate from '../components/TimeEstimate';
+import PaymentForm from '../components/PaymentForm';
+import Review from '../components/Review';
+import Header from "../components/Header";
 import "./Order.css";
 import { useLocation } from 'react-router-dom'
 
-
 function App() {
```

Comment:
I think function names for pages/components should be consistent with the file name

Reply:
I could change it, but we actually use App on all the other pages, so I wanted to keep it consistent with that.